static/app/utils/discover/charts.tsx

```diff
@@ -85,14 +87,15 @@ export function axisLabelFormatterUsingAggregateOutputType(
   type: string,
   abbreviation: boolean = false,
   durationUnit?: number,
-  rateUnit?: RateUnit
+  rateUnit?: RateUnit,
+  decimalPlaces?: number
```

Comment:
```suggestion
  decimalPlaces: number = 0
```
nit: You can default this parameter to 0 so you don't have to check for undefined below